### ISSUE TITLE
Symbols not working

### ISSUE THREAD

**Issue by martindsouza** [zabel-xyz/plsql-language#61]
In the latest release of VSC (1.29.1) Symbols doesn't work correctly when `end` is used out of context. Ex:

```sql
procedure dummy
begin
  logger.log('end');
end;

-- This won't show up
procedure another_one
begin
  null;
end;
```

In the above example only `dummy` shows up in Symbols. If we remove the line `logger.log('end');` both `dummy` and `another_one` show up in Symbols.

Another example:

```sql
procedure dummy
begin
    $if $$no_op $then
      null;
    $else
       null;
    $end
end;

-- This won't show up
procedure another_one
begin
  null;
end;
```

In this case the Oracle conditional compilation line `$end` prevents `another_one` from showing up.

Worked with @fuzziebrain to tried to fix in `syntaxes/plsql.tmLanguage` to expand some of the regular expressions but we couldn't get one working. The only `match` that we could find that detected `end` in our case was `keyword.other.sql.oracle`

**Comment by zabel-xyz** [comment]
The problem is not in .tmLanguage, which is only used for colorization.
I think this is the same issue as #58. The regExpParser must be fixed...
(Perhaps I'll have some time to fix this at the end of this week)

**Comment by fuzziebrain** [comment]
This regular expression in `main/out/src/lib/regEx/RegExParser.js` works for me:

```javascript
RegExpParser.regJumpEnd = `(\\bbegin|case\\b)|(?:(\\send\\s*.*;)\\s*(?:\\b(if|loop|case)\\b)?)`;
```

However, conditional compilation statements in the declarative part still causes a problem.

**Comment by zabel-xyz** [comment]
Fix in version 1.6.3
I added some tests, see [xyz_myPackage.sql](https://github.com/zabel-xyz/plsql-language/blob/master/main/test/sql/xyz_myPackage.sql)

**Comment by fuzziebrain** [comment]
Thanks @zabel-xyz . Issue looks resolved on my end;
